packages/strapi-plugin-documentation/services/Documentation.js

```diff
@@ -1473,6 +1483,20 @@ module.exports = {
     }
   },
 
+  /**
+   * Refer to https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#dataTypes
+   * @param {String} type
+   * @returns {String}
+   */
+  getFormat: type => {
+    switch (type) {
+      case 'datetime':
+        return 'date-time';
+      default:
+        return type;
+    }
+  },
```

Comment:
Can you make the handling of `date` and `password` explicit please? And make the default value `undefined` as the function would be wrong for other types.

Something like:
```
  getFormat: type => {
    switch (type) {
      case 'date':
        return 'date';
      case 'datetime':
        return 'date-time';
      case 'password':
        return 'password';
      default:
        return undefined;
    }
  },
```